fishjam/media_events/peer/peer.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package fishjam.media_events.peer;
+
+import "fishjam/media_events/shared.proto";
+
+// Defines any type of message sent from Peer to Membrane RTC Engine
+message MediaEvent {
+  // SCHEMAS
+  message VariantBitrates {
+    media_events.Encoding rid = 1;
```

Comment:
1. Should we change `Encoding` to `Variant` or `Variant` to `Encoding` throughout the entire project?
2. Should we describe what `rid` is or explain why we are using this name instead of `encoding` or `variant`?
3. Is it possible to rename `rid` to `encoding` or `variant`?